pkg/kong/kong.go

```diff
@@ -6,19 +6,15 @@ import (
 	"errors"
 	"net/http"
 	"path"
-	"time"
 
+	"github.com/mynameismaxz/acme2kong/pkg/httpRequest"
 	"github.com/mynameismaxz/acme2kong/pkg/logger"
 )
 
 const (
 	certificates = "certificates"
```

Comment:
constant should be Capital-letter first like Certificate.

But in this case, I think it will be path. Should be `CertificateRoute` or `CertificateUrl`